### ISSUE TITLE
[BUG] Add instructions for creating runtime fields from Timeline 

### ISSUE THREAD

**Issue by nastasha-solomon** [elastic/security-docs#2182]
The [Timeline](https://www.elastic.co/guide/en/security/master/timelines-ui.html) docs are currently missing steps for creating a runtime field from the Timeline table. Can model the steps after what's doc'd in the [Add runtime fields to detection alerts](https://www.elastic.co/guide/en/security/master/alerts-runtime-fields.html) topic. 

* NOTE: Currently there's some inconsistency in whether the "Create field" button appears in the fields browser modal. This is being addressed in https://github.com/elastic/kibana/issues/135827.

* Additional contexts where fields browser and create runtime fields are available: Hosts/Users pages -> Events, alerts table in rule details page

James S. pointed out where to access the functionality for creating runtime fields in Timeline:

<img src="https://user-images.githubusercontent.com/79124755/177344653-87dea117-3e44-4376-87cd-cb30bd20786d.jpg" width="33%">

**Comment by joepeeples** [comment]
A few items:
- We've noticed some inconsistency in when the fields browser modal includes or doesn't include the "Create field" button. Investigating further...
- Additional contexts where fields browser and create runtime fields are available: Hosts/Users pages -> Events, alerts table in rule details page

**Comment by joepeeples** [comment]
Thanks for the heads-up @nastasha-solomon. So far I haven't been able to observe the fix by spinning up a cloud deployment (ESTEC with 8.3.2-SNAPSHOT), but it might just not have gotten pushed to cloud yet. I'll try again later.

**Comment by semd** [comment]
Hello, yes we tried to push the fix into `8.3.2`, but unfortunately, it didn't make it. 
However, there will be an `8.3.3`, which will contain the bug fix, I changed the PR tag. 
It is also already fixed in `main` btw.

**Comment by nastasha-solomon** [comment]
@semd thanks for the update! Based on this new information, we might need to also need to update the 8.3.2 release notes to remove #135842. I added the PR to the release notes the day before 8.3.2 GA'ed (~6 days ago) because the PR had been merged and labeled to be doc'd for the 8.3.2 release notes. I can file a doc bug/PR to remove #135842 from the release notes today if needed.

cc: @michaelolo24 @elastic/security-docs
